ui/ducks/app/app.ts

```diff
@@ -103,7 +182,7 @@ export default function reduceApp(state = {}, action) {
     case actionConstants.SET_SMART_TRANSACTIONS_ERROR:
       return {
         ...appState,
-        smartTransactionsError: action.payload,
+        smartTransactionsError: action.value,
```

Comment:
Whereas this one is called from the actions.js file with payload, but it was changed to use value?

Reply:
fixed now 🙏